### PR TITLE
revived RPi optimization option

### PR DESCRIPTION
**Pull request by 66eli77**
this PR should fix the issues #3272 and #2695

### CODE REVIEW

File: kalite/contentload/management/commands/unpack_assessment_zip.py [hunk 1]
```diff
@@ -52,14 +52,20 @@ def handle(self, *args, **kwargs):
 
         if is_valid_url(ziplocation):  # url; download the zip
             logging.warn("Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time...")
-            r = requests.get(ziplocation)
-            r.raise_for_status()
-            f = StringIO.StringIO(r.content)
+            #this way we can download stuff larger than the device's RAM
+            r = requests.get(ziplocation, prefetch=False)
+            f = open("tobezipped", "w+")
```
Comment (aronasorman) [inline kalite/contentload/management/commands/unpack_assessment_zip.py]: Just use `tempfile.TemporaryFile`.

**Comment by aronasorman** [comment]
Can you tag the issues being solved here in the PR description @66eli77?

**Comment by aronasorman** [comment]
Comments still not addressed.

File: kalite/contentload/management/commands/unpack_assessment_zip.py [hunk 2]
```diff
@@ -52,14 +53,19 @@ def handle(self, *args, **kwargs):
 
         if is_valid_url(ziplocation):  # url; download the zip
             logging.warn("Downloading assessment item data from a remote server. Please be patient; this file is big, so this may take some time...")
-            r = requests.get(ziplocation)
-            r.raise_for_status()
-            f = StringIO.StringIO(r.content)
+            #this way we can download stuff larger than the device's RAM
```
Comment (aronasorman) [inline kalite/contentload/management/commands/unpack_assessment_zip.py]: Space after the '#'

Comment (66eli77) [inline kalite/contentload/management/commands/unpack_assessment_zip.py]: as you wish

**Comment by aronasorman** [comment]
Other than that, this looks good.

**Pull request merged**